index.html

```diff
@@ -8,6 +8,37 @@
     <title>Personal Portfolio</title>
 </head>
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
+
+        <figure class="menu-icon">
+            <img src="./images/Union.png" alt="">
+        </figure>
+    </nav>
+
+    <section class="main">
+        <header class="header">
+            <div class="banner">
+                <h1>I’m Simo Nkosi <br>
+                    Glad to see you!
+                </h1>
+                <p>I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <div class="social-media-icons">
+                    <a href="http://twitter.com"><img src="./images/twitter.svg" alt="" class="social-icon"></a>
+                    <a href="http://Linkedin.com"><img src="./images/Linkedin-icon.svg" alt="" class="social-icon"></a>
+                    <a href="http://medium.com"><img src="./images/medium.svg" alt="" class="social-icon"></a>
+                    <a href="http://github.com"><img src="./images/github.svg" alt="" class="social-icon"></a>
+                    <a href="http://angellist.com"><img src="./images/angellist.svg" alt="" class="social-icon"></a>
+                </div>
```

Comment:
**REQUIRED:** You should use `ul` to group your icons, and `li` per icon. The main reason of is that you should practice semantic HTML as much as possible.